### ISSUE TITLE
Provide an option to specify HTTP Protocol version

### ISSUE THREAD

**Issue by asarkar** [a85/POSTMan-Chrome-Extension#6242]
https://github.com/postmanlabs/postman-app-support/issues/876 was closed with a passing comment. Postman now has a desktop version, so what the browser does is irrelevant.

**Comment by shamasis** [comment]
@asarkar - indeed! that was quite some time back. You'll be pleased to know that work is in progress for more low-level control over the request. Stay tuned.

**Comment by cassiomolin** [comment]
Any updates on this?

**Comment by Dysar** [comment]
What is the status of the feature?

**Comment by Hell4Ge** [comment]
Anything?

**Comment by amir1376irani** [comment]
If you check postman roadmap board, you'll see this issue is in later cards!
Are you guys serious about it? @postmanlabs
it's a basic thing that many devs need this

In the time that we hear news about QUIC and http/3 (even there are implemented like proxygen C++ library by Facebook)... postman still doesn't support http/2 for requests??
Man there's just few days to 2020 and we can just test our codes over http/1.1 protocol with postman latest version (7.14.0) !!

**Comment by cassiomolin** [comment]
> You'll be pleased to know that work is in progress for more low-level control over the request. Stay tuned.

We surely have been tuned and haven't received any updates so far. Is there at least an ETA for this?

**Comment by coxrichuk** [comment]
+1 for http/2

**Comment by Mexicoder** [comment]
Any updates on this?

**Comment by LeaderXFX** [comment]
+1 for http/2

**Comment by xipho** [comment]
+1 fro http/2

**Comment by lejiend** [comment]
+1 for enable specifying http protocol version

**Comment by arlemi** [comment]
This issue is about giving the option to specify the protocol, not the implementation itself. Any news regarding support for HTTP2 will be given in #2701.

**Comment by jvbutera** [comment]
Any update on this request?

**Comment by asarkar** [comment]
> Stay tuned.

Good thing you didn’t ask me to hold my breath.

**Comment by shahargl** [comment]
+1 for http/2

**Comment by devilHorns13** [comment]
One good practice is to enable HTTP/1 support on the server-side when testing APIs internally.

**Comment by xipho** [comment]
> One good practice is to enable HTTP/1 support on the server-side when testing APIs internally.

It's not a good practice in my opinion. What if I want to test difference between HTTP/1.1 and HTTP/2.0 behaviour? Or test migration to HTTP/2.0 from old HTTP/1.1?

**Comment by aaronmfparr** [comment]
We could really use this feature.

**Comment by HWiese1980** [comment]
Wassup? Still no update? I could use this right now! Why is this issue closed? It is not solved yet!

**Comment by mhavelka77** [comment]
Still waiting for this

**Comment by Shimanshu83** [comment]
any update on this?

**Comment by codenirvana** [comment]
Very soon! 🤐

<img width="616" alt="image" src="https://github.com/user-attachments/assets/106cbb5c-8ab4-41e4-97ba-9f26906c0e99">

**Comment by parthverma1** [comment]
Hi everyone,
With the resolution of https://github.com/postmanlabs/postman-app-support/issues/2701 available in v11.8 ([Download Link](https://www.postman.com/downloads/)), we have added a version selector for HTTP requests. 

![641f0d5a-d9c9-4cb4-b419-4ed6c319e67f-ezgif com-optimize](https://github.com/user-attachments/assets/78203a98-dbdf-4efd-b2cc-55723dae7cae)

You can also inspect the HTTP version used for a request using network tooltip.

With this launch, we will be closing this issue. We encourage users to open a new issue if you encounter while using different HTTP versions.

**Comment by asarkar** [comment]
@parthverma1 I’m the author of this ticket. Can you please explain how’d the user specify a version as 1.0, 1.1 or 1.2? The animation shows a generic bucket for all 1.x, which isn’t really providing an ability to choose a version.

**Comment by parthverma1** [comment]
With the introduction of HTTP/2, we added support for specifying HTTP/1.x and HTTP/2. We believe that this addresses bulk of the user's comments on this ticket.

I have opened up a new issue to track the ability of setting HTTP/1 minor versions as a separate feature request. https://github.com/postmanlabs/postman-app-support/issues/13074

I would request you to comment on that ticket. I would also appreciate sharing how you would want to use this feature, specific use cases that are blocked by the absence of this feature. This will help us gather community feedback, understand what is expected by the feature request and prioritize this feature on our roadmaps accordingly.
